tests/test_live_api.py

```diff
@@ -75,15 +75,26 @@ def test_get_itinerary(self):
 
     def test_get_image_url(self):
         url = self.dossier.get_map_url()
-        self.assertIsInstance(url, basestring)
+
+        if sys.version_info.major < 3:
```

Comment:
It might be nice to wrap this in a simple function -- That, or does `six` provide a utility here? Not huge on all this boiler plate sprinkled in gapipy.